### ISSUE TITLE
Notification only on mention?

### ISSUE THREAD

**Issue by poVoq** [jsxc/jsxc#1074]
Maybe I am overlooking something, but I seem to be getting a notification for every message, not only those that mention my nick.

Especially on busy MUCs that results in hundreds of notifications upon opening JSXC up to the point of crashing the KDE desktop notification manager it seems.

**Comment by sualko** [comment]
> I seem to be getting a notification for every message, not only those that mention my nick.

That's right, but I see your problem. A general and room specific configuration would be nice.

> up to the point of crashing the KDE desktop

Maybe we should handle the case of mass notification separately. Using a counter to not show a notification or to modify an existing notification to display a summery.
